.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "3.6"
+install:
+  - pip install nose twine
```

Comment:
Why nose ?
Why not use develop_dependencie in the setup.py ?
If you still want to use nose, you can say it in the setup.py so you can run `./setup.py test`

Reply:
I could remove nose and just use unittest.